geode-core/src/main/java/org/apache/geode/distributed/LocatorLauncher.java

```diff
@@ -1848,8 +1884,7 @@ public LocatorLauncher build() {
     private final String name;
 
     Command(final String name, final String... options) {
-      assert !StringUtils
-          .isBlank(name) : "The name of the locator launcher command must be specified!";
+      assert !isBlank(name) : "The name of the locator launcher command must be specified!";
```

Comment:
As long as we're touching this, we could consider using `isNotBlank` for better readability.